### ISSUE TITLE
MIHO013 eTRV: VALVE_STATE

### ISSUE THREAD

**Issue by genestealer** [Achronite/node-red-contrib-energenie-ener314rt#65]
I cannot see when/which commands are needed to make the MIHO013 eTRV will return the "VALVE_STATE", can you please confirm?

**Comment by Achronite** [comment]
Yeah, as far as I know it is never returned by the eTRV.  You can find out what is sent back (and when) here: https://github.com/Achronite/node-red-contrib-energenie-ener314rt#etrv-monitor-messages

**Comment by genestealer** [comment]
That is really frustrating, so we need to assume "VALVE_STATE" is as we last set, no way to check it. We may be able to infer though, as I 'think' the 5 mins returned data of a "closed/open" valve is different to a valve set to "auto". Will need to check.

**Comment by Achronite** [comment]
@genestealer Did you get any further with this?  In [mqtt-energenie-ener314rt](https://github.com/Achronite/mqtt-energenie-ener314rt) I optimistically update the state topic based on command after receipt.

I noticed you also have the MiHome gateway - are you seeing anything being sent/returned to the eTRV (in debug logs) that I have not documented?

**Comment by genestealer** [comment]
Hi @Achronite ,
I'm no longer using the MiHome gateway, it's completely unplugged.

Regarding state optimism, I am trying the approach of not retaining the values sent via MQTT, but allowing Home Assistant to "remember" what was last sent (E.G. see REPORTING_INTERVAL below)

![image](https://user-images.githubusercontent.com/5207490/215290518-b3d18c0b-c33b-4e6c-8c35-0265d1b18274.png)

eTRV after setting interval
![image](https://user-images.githubusercontent.com/5207490/215290529-e279619b-76a5-4fb9-98f0-2dead0bf8363.png)


eTRV with interval never set.
![image](https://user-images.githubusercontent.com/5207490/215290562-39b179c1-1b56-4f73-b7b9-73af33031b90.png)

**Comment by Achronite** [comment]
I have checked again with energenie, here is their response:
> Hi Achronite,
>
>The eTRV does not provide a confirmation of receipt for target temperatures or valve state.  The valve state when set up will be normal operation and as each valve type will have a slight difference in stem length, the valve does not full open or close unless a specific command is sent, or if there is a large enough difference between the target temperature and the current temperature.

So I'm going to close this as it cannot be done.

**Comment by genestealer** [comment]
Thank you for reaching out to their support to confirm this. It's good to know that they are still actively responding to technical queries. It would be also really interested to know if there was some sort of special command that we could send that would increase the polling rate to something less than 5 minutes or is that a question that's already been asked? I was waiting 5 minutes for it to respond can be quite frustrating when doing debugging.

**Comment by Achronite** [comment]
I think I got the 5 minutes from the energenie docs as the minimum wait. SET_REPORTING_PERIOD is the command that changes this; you could try using it to change the wakeup period to less than 300 and see what happens?

From docs: 
_To  save power and extend battery life, the eTRV communicates with the  Gateway once every 5 minutes.  It will not begin to respond to a command  for those 5 minutes.  Once it has received the command, the eTRV will  slowly make adjustments to the radiator valve to adjust to the new  target temperature.  Wait 30 minutes to confirm whether the new target  temperature has been received._
